GitCommands/Remotes/AzureDevOpsRemoteParser.cs

```diff
@@ -1,14 +1,22 @@
 using System.Diagnostics.CodeAnalysis;
+using System.Text.RegularExpressions;
 
 namespace GitCommands.Remotes
 {
-    public sealed class AzureDevOpsRemoteParser : RemoteParser
+    public sealed partial class AzureDevOpsRemoteParser : RemoteParser
     {
-        private static readonly string VstsHttpsRemoteRegex = @"^https:\/\/(?<owner>[^.]*)\.visualstudio\.com\/(?<project>[^\/]*)\/_git\/(?<repo>.*)$";
```

Comment:
Unrelated to this PR, but git-word-diff really helps here:
![image](https://github.com/gitextensions/gitextensions/assets/6248932/70d85f53-7b9e-4380-b532-e39a19f09558)

Reply:
![image](https://media.tenor.com/SWKI18Zs_H8AAAAM/magic-meme.gif)